docs/appendix/glossary.md

```diff
@@ -54,7 +54,7 @@ The Glean SDK provides default pings and allows for custom ping, see [Glean Ping
 
 ## Submission
 
-"To submit" means to collect & to upload a ping.
+"To submit" means to collect & to en-queue a ping for uploading.
```

Comment:
I don't think `en-queue` should have a hyphen, [according to the dictionary](https://www.lexico.com/definition/enqueue)

Reply:
Me either, but our syntax checker complained about that. There is no discussing with the dictionary though, I'll see what I can do about the syntax checker.

Reply:
You can add the word to the [dictionary file](https://github.com/mozilla/glean/blob/main/.dictionary) :)